plugins/node/opentelemetry-instrumentation-express/examples/package.json

```
@@ -0,0 +1,50 @@
+{
+  "name": "express-example",
+  "private": true,
+  "version": "0.23.0",
```

```suggestion
  "version": "0.28.0",
```
I guess this should go in sync with express instrumentation itself